cachi2/core/package_managers/yarn/main.py

```diff
@@ -1,4 +1,5 @@
 import logging
```

Comment:
**Commenting on the commit message:**
You're missing any kind of commit message. Unless being a trivial change, the change should be described and explained in the commit message so that an 'external' contributor understands why we actually need the change and why the commit did it the way it had done it (unless it's truly a trivial change). There's no abstract accompanying the PR, there's only a vague reference in your branch naming hinting at STONEBLD-1788 which doesn't tell much external contributors.
I would suggest describing in the commit message what "regular workflow" is; Note that many people look and search primarily within commit history first before scrolling through the list of pull requests.

Also, naming the commit subject as "Fetch dependencies for regular workflow" doesn't say much about what the commit is about at all unless diving straight into the code, because the subject should IMO follow some logical structure, .e.g. 'package_managers: yarn: Fetch dependencies ...' which immediately hints at which part of the code the commit is touching.

Reply:
I think the docs should handle proper descriptions. Also, I would expect that any external contributor would dive into the code at first and then start to look up the commit history or not ?

Reply:
which docs? If you're referring to the yarnberry docs then you should at least link it and not expect people go figure out the corresponding section on their own.

Sure, but imagine a block of code changing a 100x. How would you know the complete history and reasoning behind a change if there's no commit message explaning it? The thing is unfortunately, you can't make assumptions on what people will or will not do, that's why we have commit history and commit messages -> for proper bookkeeping and reminding us of past decisions.

Reply:
The docs for yarn do not exist yet. I could add a link to yarn documentation, but again I would expect an external contributor to have some knowledge about that. :smile: 

Anyway I updated the commit messages to be more verbose.